appcelerator/titanium_mobile
# Android: Fixed picker spinner to size/position columns correctly on Android 5.0 and higher

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-19822

**Test:**
1. Build and run the `SpinnerTest.js` file attached to [TIMOB-19822](https://jira.appcelerator.org/browse/TIMOB-19822) on Android 5.0 or higher.
2. Verify that all spinner columns have the same height.
3. Verify that the item selected in each spinner column is correctly centered vertically.
4. See `SpinnerTest-good.png` attached to [TIMOB-19822](https://jira.appcelerator.org/browse/TIMOB-19822) for an idea of what spinners should look like.
5. Re-run the test on an Android 4.x device. (This is to verify that there are no regressions.)
